#Position of a robot
int64 id
geometry_msgs/Pose2D pose
time time
